java/trace/src/main/scala/org/typelevel/otel4s/java/trace/WrappedSpanContext.scala

```diff
@@ -42,6 +43,12 @@ private[java] final case class WrappedSpanContext(
   lazy val traceFlags: TraceFlags =
     TraceFlags.fromByte(jSpanContext.getTraceFlags.asByte)
 
+  lazy val traceState: TraceState = {
```

Comment:
how difficult would it be to wrap `JTraceState` instead?

Reply:
We should unseal `TraceState` then. Overall, it's possible and doable, but the implications could be clearer. 

We can experiment with a `delegate` instance. I used it with `SpanContext` while sketching sdk-trace module: https://github.com/typelevel/otel4s/blob/17265ca6f9a6725ca2a49e741cda8f839a9eeb0c/core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanContext.scala#L245C1-L261C6.

But I would keep it outside of the current PR.

Reply:
I didn't even notice that it was sealed, oops